Dockerfile

```diff
@@ -24,12 +24,6 @@ RUN curl -L https://mirror.openshift.com/pub/openshift-v4/clients/rosa/latest/ro
     && chmod +x /usr/bin/rosa \
     && rosa version
 
-# Install the OpenShift CLI (OC)
-RUN curl -L https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz --output /tmp/openshift-client-linux.tar.gz \
```

Comment:
Why are we removing oc binary this?
i thought we were going to remove rosa

Reply:
Oc binary should be always downloaded from the cluster under test. Otherwise you are always testing with latest stable oc, irrespective of openshift version on your cluster.  All these binaries increases the image size too. If you look at quay, our latest tag is 807.9 MiB, and removing just oc reduced it to 668MiB. I will be removing rosa as well.